shell/platform/tizen/tizen_vsync_waiter.cc

```diff
@@ -85,7 +107,6 @@ bool TizenVsyncWaiter::CreateTDMVblank() {
     return false;
   }
 
-  tdm_client_vblank_set_enable_fake(vblank_, 1);
```

Comment:
I rember we add this line for requesting vblank successful in sleep mode.

Reply:
Yes, I remember that. but I'm not sure, I don't think it's necessary anymore. Could you test it?

Reply:
https://github.sec.samsung.net/f-project/home/issues/39
If you verify this case,that's fine.

Reply:
If you are concerned that this will cause a new problem, I will revert this change. Does not matter.

Reply:
I am building your code and test it. if no problem, we don't need add it.

Reply:
guanxw@yuanzw:~/Tizen/xiaowei.guan/gallery$ flutter-tizen run
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
Warning
──────────────────────────────────────────────────────────────────────────────
Your Flutter application is created using an older version of the Android
embedding. It's being deprecated in favor of Android embedding v2. Follow the
steps at

https://flutter.dev/go/android-project-migration

to migrate your project.
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

Using hardware rendering with device Tizen M-6.0-x86. If you notice graphics artifacts, consider enabling software rendering with "--enable-software-rendering".
Launching lib/main.dart on Tizen M-6.0-x86 in debug mode...
The active profile is used for signing.
Building a Tizen application in debug mode...                      65.1s
✓ Built build/tizen/io.flutter.demo.gallery-2.6.0.tpk (97.2MB).
Installing build/tizen/io.flutter.demo.gallery-2.6.0.tpk...         5.5s
I/ConsoleMessage( 5429): ../../flutter/shell/platform/tizen/tizen_embedder_engine.cc: RunEngine(161) > FlutterEngineRun Success!
E/ConsoleMessage( 5429): ../../flutter/shell/platform/tizen/tizen_vsync_waiter.cc: HandleVblankLoopRequest(130) > ERROR, ret = -10

when in sleep mode, we can't remove this line code

Reply:
Oh, I understood at this time. did you mean to use flutter-tizen to relaunch the app in sleep mode?
This is a situation I have not considered. Let's take a look again. Thanks for the detailed explanation.

Reply:
I've seen the difference between before and after and eventually I reverted this change
@xiaowei-guan  Thank you for your meticulous review.